var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -140,7 +140,7 @@ class Cmake(Package):
     variant('ownlibs', default=True,  description='Use CMake-provided third-party libraries')
     variant('qt',      default=False, description='Enables the build of cmake-gui')
     variant('doc',     default=False, description='Enables the generation of html and man page documentation')
-    variant('openssl', default=True,  description="Enables CMake's OpenSSL features")
+    variant('openssl', default=True,  description="Enable openssl for curl bootstrapped by CMake when using +ownlibs")
```

Comment:
I could be wrong but should the default be `false` then?

Reply:
There's a commented conflict somewhere down here to fix it to a specific value. The actual value is irrelevant because it's unused, I think True can also make sense (*if* curl is bootstrapped then use ssl -- but it's not bootstrapped). Adding a conflict could still be good though